copy_with_style.js

```diff
@@ -175,33 +177,42 @@ class Copy_With_Style {
 		this.observe = observe;
 		this.show_progress = progress ? true : false;
 		this.copy_wrapper = copy_wrapper;
-		this.class_button = class_button,
-		this.class_preparing = class_preparing,
-		this.class_ready = class_ready,
-		this.deep_exclusions = deep_exclusions,
-		this.shallow_exclusions = shallow_exclusions,
-		this.extra_deep_exclusions = extra_deep_exclusions,
-		this.extra_shallow_exclusions = extra_shallow_exclusions,
-		this.debug = debug;
+		(this.class_button = class_button),
+			(this.class_preparing = class_preparing),
+			(this.class_ready = class_ready),
+			(this.deep_exclusions = deep_exclusions),
+			(this.shallow_exclusions = shallow_exclusions),
+			(this.extra_deep_exclusions = extra_deep_exclusions),
+			(this.extra_shallow_exclusions = extra_shallow_exclusions),
+			(this.debug = debug);
```

Comment:
Not sure what your editor did here or why. Looks odd, in that it selectively seems to have put some of the options in brackets? Worth undoing?

Reply:
Prettier did this, also the most popular linting tools, I think we can change the `,` to `;`